roofit/roofitcore/src/RooAbsCollection.cxx

```diff
@@ -929,6 +929,8 @@ RooAbsArg * RooAbsCollection::find(const char *name) const
 
   if (_hashAssistedFind || _list.size() >= _sizeThresholdForMapSearch) {
     if (!_hashAssistedFind || !_hashAssistedFind->isValid()) {
+      if (_hashAssistedFind)
+         delete _hashAssistedFind;
```

Comment:
The if statement is redundant, the exact same if statement is inside the operator delete implementation.

Reply:
Yes, we have removed that in Tomas' follow up PR